x-pack/legacy/plugins/ml/server/models/data_frame/transforms.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { callWithRequestType } from '../../../common/types/kibana';
+import {
+  DATA_FRAME_TRANSFORM_STATE,
+  DataFrameTransformEndpointData,
+} from '../../../public/data_frame/pages/transform_management/components/transform_list/common';
+
+export function transformServiceProvider(callWithRequest: callWithRequestType) {
+  async function deleteTransform(transformId: string) {
+    return callWithRequest('ml.deleteDataFrameTransform', { transformId });
+  }
+
+  async function stopTransform(options: {
+    transformId: string;
```

Comment:
could be `transformId: DataFrameTransformId`

Reply:
Updated in f7a9d5de55631c0e03cb43586efbddc0b2a15555